app/config/FrontendAppConfig.scala

```diff
@@ -70,6 +70,7 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig) {
   lazy val govUKHelp: String = servicesConfig.getString("urls.footer.govukHelp")
 
   def languageTranslationEnabled: Boolean = servicesConfig.getBoolean("features.welshLanguage")
+  lazy val isExternalTest: Boolean = servicesConfig.getBoolean("features.submission")
```

Comment:
naming convention confusing: when `features.submission` is set to `true`, then the `isExternalTest` is `true` - maybe rename this variable to `isNotExternalTest` and its usage accordingly?